modules/auth_email/module.json

```diff
@@ -0,0 +1,91 @@
+{
+	"name": "Auth Email",
+	"description": "Authenticate users with email only or an email/password combination.",
+	"icon": "key",
+	"tags": [
+		"core",
+		"auth",
+		"user"
+	],
+	"authors": [
+		"rivet-gg",
+		"NathanFlurry",
+		"Blckbrry-Pi"
+	],
+	"status": "stable",
+	"dependencies": {
+		"email": {},
+        "identities": {},
+        "users": {},
+		"tokens": {},
+		"user_passwords": {},
+		"rate_limit": {}
+	},
+	"defaultConfig": {
+		"enable": {
+			"withPassword": true,
+			"passwordless": true,
+			"linking": true
+		},
+		"fromEmail": "hello@test.com",
+		"fromName": "Authentication Code"
+	},
+	"scripts": {
```

Comment:
repost: this api is suuuper confusing (i know i helped design it :P). thoughts on splitting this up in to two modules which expose public scripts that call internal scripts exposed in this module. so devs will call auth_email_password and auth_email_passwordless.